runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IndustriesSection/IndustriesSection.tsx

```diff
@@ -0,0 +1,25 @@
+import useTranslations from '#src-app/hooks/useTranslations';
+import Typography from '#src-landing/components/Typography';
+
+import { SLIDES } from '#src-landing/views/sections/IndustriesSection/IndustriesSection.utils';
+
+import IndustriesCarousel from './IndustriesCarousel/IndustiresCarousel';
+import styles from './IndustriesSection.module.scss';
+
+const IndustriesSection = () => {
+    const { translate } = useTranslations();
+    return (
+        <section className={styles.root}>
+            <div className={styles.title}>
+                <Typography variant="h2">
+                    <div>{translate('Landing.Industries.Title.Part.1')}</div>
```

Comment:
one translate text in div, another not? div inside Typography? just saying to take a second thought.

Reply:
yeah, this looks ugly and stupid buuuuut, we achive line breake without <br /> because div is a block element. also i do it all inside one typography so this section has one h2 and not two h2's

Reply:
makes sense